xml/install_caasp_heat_templates.xml

```diff
@@ -147,7 +147,7 @@ git clone https://github.com/SUSE/caasp-openstack-heat-templates
    checking that the <literal>provisioning_status</literal> changes to
    <literal>Active</literal>.
   </para>
-  <screen>&prompt.ardana;openstack loadbalancer show
+  <screen>$ openstack loadbalancer show
```

Comment:
ditto to what Dima said in the other review, you can use the &prompt.user; entity